css/base/rutorrent/rutorrent-base.css

```diff
@@ -1804,3 +1804,7 @@ div#dlgEditRules div.dlg-header {
 div#dlgEditFilters div.dlg-header{
     background-image:none;
 }
+
+div.label-count{
+    background-color: var(--modal-bg-color);

```

Comment:
Dont use the modal var. Use the `--accent-color` var and the `--label-text-color` var for text

Reply:
I tried but it doesn't work with Nord for example. 
Its the same color as the background so the tag is not readable as expected. --modal-bg-color is the best compromise I've found. Otherwise you should add a --tag-color or something?

Reply:
It's not the same, I use it in all themes. Look at the variable, it's not a hex value. Look in the other bases how it's used.

Reply:
Result with var(--modal-bg-color);
![image](https://user-images.githubusercontent.com/35723598/217742531-b3623e40-e97e-46fc-8c06-9452c158e2cb.png)

Result with rgb(var(--accent-color)) for bg 
![image](https://user-images.githubusercontent.com/35723598/217742838-a95398b6-52b3-4c3e-8052-db627099412d.png)

Result with rgb(var(--accent-color)) for bg + var(--label-text-color) for text
![image](https://user-images.githubusercontent.com/35723598/217743136-b6865099-0a9b-4d5e-9738-ed5ed072c1e8.png)

I understand it might be the expected way of using the color palettes, thank you for the hint, didnt see that only one wasn't an hex.
Nevertheless, it's not nice in term of design. Either not readable, either it makes the tags more catchy than the label itself... which shouldn't be the case.
I can update the PR so it solves people issues but I will stick with my custom style, I can't use this.

Reply:
Imo it's the text shadow that makes it look wierd. You can try and remove that.

Reply:
I changed it, better but accent color is still putting the focus on this tags and not the label text, but its cleaner so feel free to merge it ;)